NDTensors/src/lib/TypeParameterAccessors/src/Base/iswrappedarray.jl

```diff
@@ -0,0 +1,80 @@
+using SimpleTraits
```

Comment:
```suggestion
using SimpleTraits: @traitdef, @traitimpl, @traitfn
```

Reply:
For some reason when I don't add the entirety of `SimpleTraits` I get a weird error about Julia not having `Not`
```
julia> set_ndims(Array{Float32,2}, 3)
ERROR: UndefVarError: `Not` not defined
Stacktrace:
 [1] trait
   @ NDTensors.TypeParameterAccessors ~/.julia/packages/SimpleTraits/l1ZsK/src/SimpleTraits.jl:171 [inlined]
 [2] set_ndims(type::Type{Matrix{Float32}}, ndim::Int64)
   @ NDTensors.TypeParameterAccessors ~/.julia/packages/SimpleTraits/l1ZsK/src/SimpleTraits.jl:331
 [3] top-level scope
   @ REPL[1]:1
   ```

Reply:
What about doing:
```julia
using SimpleTraits: @traitdef, @traitimpl, @traitfn, Not
```
I assume what's happening is that when you use negated traits like `!IsWrappedArray` the macro wraps it in the `Not` type.

Reply:
For now let's just write the code how we think is best, and by the end if there are warnings we can look into them more carefully.

Reply:
This is updated now so I am resolving this conversation